src/theme/default/index.js

```diff
@@ -18,7 +28,7 @@ export const defaultTheme = {
   },
   lineHeights: {
     body: 1.5,
-    heading: 1.25,
+    heading: 1.3,
```

Comment:
muzu jeste poprosit pridat `1.3` pro `buttons`

Reply:
Máš to tam.

Reply:
Bylo by možné přidat rovnou i `border-radius: 8px`? 🙏

Reply:
Máš ho tam